src/runtime/client.ts

```diff
@@ -0,0 +1,30 @@
+import { Chain, createPublicClient, http } from 'viem'
+import {
+  arbitrum,
+  arbitrumSepolia,
+  celo,
+  celoAlfajores,
+  mainnet,
+  optimism,
+  optimismSepolia,
+  sepolia,
+} from 'viem/chains'
+import { NetworkId } from '../api/networkId'
+
+const networkIdToViemChain: Record<NetworkId, Chain> = {
+  [NetworkId['celo-mainnet']]: celo,
+  [NetworkId['ethereum-mainnet']]: mainnet,
+  [NetworkId['arbitrum-one']]: arbitrum,
+  [NetworkId['op-mainnet']]: optimism,
+  [NetworkId['celo-alfajores']]: celoAlfajores,
+  [NetworkId['ethereum-sepolia']]: sepolia,
+  [NetworkId['arbitrum-sepolia']]: arbitrumSepolia,
+  [NetworkId['op-sepolia']]: optimismSepolia,
+}
+
+export function getClient(networkId: NetworkId) {
+  return createPublicClient({
+    chain: networkIdToViemChain[networkId],
+    transport: http(),
+  })
+}
```

Comment:
Does this erase the actual typed chain? Not sure it matters though.

Reply:
yeah I'm not sure either

Reply:
I tinkered around with it and didn't come up with any useful improvements

Reply:
the best I thought of was setting the return type to `PublicClient<ReturnType<typeof http>, Chain>` explicitly, but that still lacks specificity on the chain type, and idk if it would help us

Reply:
my sense is that the more generic return type we'll get from getClient should throw type errors if we ever try to access some network-specific functionality that isn't present for all chains. so we'll have a chance to deal with it if it comes up. but lmk if you're thinking of something else